### ISSUE TITLE
lightning:stuff tags not highlighted in .cmp viles

### ISSUE THREAD

**Issue by mshanemc** [forcedotcom/salesforcedx-vscode#573]
### Summary

opening a lightning component since last Thursday, none of the the non-html tags are highlighted anymore 

### Steps To Reproduce:

1.  Find a repo that has a lightning component with markup, or create one
2. Open in vscode. 

### Expected result

syntax highlighting on LC.cmp
### Actual result

no highlitghting
### Additional information

_Feel free to attach a screenshot_.
<img width="1676" alt="screen shot 2018-08-16 at 2 25 41 pm" src="https://user-images.githubusercontent.com/4261788/44366046-12f41780-a491-11e8-8baa-acb66a643adf.png">

**VS Code Version**: 1.26.1

**SFDX CLI Version**: sfdx-cli/6.29.0-0a2482692c (darwin-x64) node-v8.9.4

**OS and version**: macos 10.13.6

**Comment by vazexqi** [comment]
@mshanemc - Did you mean to include a _different_ screenshot?

**Comment by mshanemc** [comment]
![screen shot 2018-08-20 at 3 52 18 pm](https://user-images.githubusercontent.com/4261788/44371377-3bd1d800-a4a4-11e8-8ed5-5db2a65d1a8d.png)

yep, sorry about that.

**Comment by daeschwed** [comment]
@mshanemc As a workaround, add the following snippet to your user settings in VS Code. That fixes the highlighting issue.

`"files.associations": { "*.cmp": "visualforce" }`

**Comment by vazexqi** [comment]
#534 might actually be related to this. If they did change the grammar, then it seems that marking `<namespace:tagname>` could cause a spike if the entire line is now errorneous.... 🤔

**Comment by erocheleau** [comment]
@ntotten The fix suggested to force the file association of *.cmp to "visualforce" has worked for me and gotten rid of the red <aura: tags.

https://github.com/forcedotcom/salesforcedx-vscode/issues/573#issuecomment-416990920

**Comment by ntotten** [comment]
@lcampos This might be a good short term fix ^. The visualforce language is using an older version of the HTML grammar & language server from before it was broken for `foo:baz` elements.

**Comment by erocheleau** [comment]
I don't know if VSCode has gotten better at letting you "extend" grammar and languages.
Last time I tried it was not possible so you had to re-do everything.

**Comment by ntotten** [comment]
Yeah, it is unfortunately not possible for the HTML language. We had to fork the HTML language server for Visualforce. We will likely have to do the same for aura to fix this problem.

**Comment by lcampos** [comment]
@ntotten unfortunately that short fix (`"files.associations": { "*.cmp": "visualforce" }`) results in losing autocompletion and other language server capabilities.

**Comment by daeschwed** [comment]
@lcampos I was merely suggesting that as a "workaround" rather than an actual fix.
I was not aware it would break things, too.
Thanks for the update.

**Comment by ntotten** [comment]
Will go out in the release this Thursday.
